simple/stats/runner.py

```diff
@@ -261,100 +270,111 @@ def _vertical_specs_dcids(self,
   def _generate_svg_cache(self):
     generate_svg_cache(self.db, self.svg_specialized_names)
 
-  # If the fh is a "special" file, append it to the self.special_handlers dict.
-  # Returns true if it is, otherwise false.
-  def _maybe_set_special_fh(self, fh: FileHandler) -> bool:
-    file_name = fh.basename()
-    file_type = self.special_file_to_type.get(file_name)
-    if file_type:
-      self.special_handlers[file_type] = fh
-      return True
-    return False
+  def _check_if_special_file(self, file: File) -> None:
```

Comment:
This method is slightly more expensive than its predecessor but should be ok in practice.